Gir.toml

```diff
@@ -155,24 +155,28 @@ status = "generate"
 [[object]]
 name = "GtkSource.FileLoader"
 status = "generate"
+version = "3.14"
```

Comment:
Please rerun `make gir` on Gir.toml changes
Also I not sure that `FileLoader` was added only in 3.14

Reply:
Puzzling: I _did_ run `make gir` before building...

GtkSourceFileLoader was added to GTK at 3.14, doesn't gtk-rs track those release numbers?  The gtk-rs doco also mentions v3_14.

Reply:
I understand why auto/mod.rs not changed: Gir already know that fileloader was added in 3.14 without this line.
Please remove version here and and in filesaver

Reply:
Does gir know it? The version annotation only seems to be on the constructor and not on the struct definition

Reply:
Yes, we checks functions https://github.com/gtk-rs/gir/blob/master/src/analysis/info_base.rs#L43